Topic_3/shoppingCart/src/main/java/com/mycompany/app/shoppingCart/Services/ShoppingCartServiceImp.java

```diff
@@ -0,0 +1,102 @@
+package com.mycompany.app.shoppingCart.Services;
+
+import java.util.List;
+
+import com.mycompany.app.shoppingCart.buisnessObject.Product;
+import com.mycompany.app.shoppingCart.buisnessObject.ProductAmount;
+import com.mycompany.app.shoppingCart.buisnessObject.User;
+import com.mycompany.app.shoppingCart.stub.DataBaseConnectorStub;
+
+public class ShoppingCartServiceImp implements ShoppingCartService {

```

Comment:
Don't use System.out.println() because this only print in console. If you develop an app for web, you lose messages.